HousingSearchApi.Tests/V1/E2ETests/Steps/GetTenureSteps.cs

```diff
@@ -41,16 +41,6 @@ public async Task WhenAPageSizeIsProvided(int pageSize)
             _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
         }
 
-        public void ThenTheLastRequestShouldBeBadRequestResult()
```

Comment:
Hey Hanna, why were those removed (Same for assets)?

Reply:
I noticed that we have the same methods for all Steps. I moved this method to the `BaseSteps` class